### PR TITLE
Numlock warning like capslock

### PR DESCRIPTION
**Pull request by razcakappa**
Fixed [#217](https://github.com/elementary/greeter/issues/217)

### CODE REVIEW

**Comment by razcakappa** [comment]
Can anyone review this changes please? At least the op?

**Comment by cassidyjames** [comment]
I will say functionally at first glance this appears to work fine, though.

**Comment by cassidyjames** [comment]
Some screenshots of this PR in action:

screenshot from 2019-02-26 15 24 42
screenshot from 2019-02-26 15 24 29
screenshot from 2019-02-26 15 24 18
screenshot from 2019-02-26 15 24 05

**Comment by danrabbit** [comment]
I think I agree with these items being next to each other in columns, but yeah packing them directly into the grid like this causes issues. They should probably live inside their own container so as not to disrupt the alignment of the other widgets

File: src/Cards/UserCard.vala [hunk 1]
```diff
@@ -49,13 +49,15 @@ public class Greeter.UserCard : Greeter.BaseCard {
         var session_button = new Greeter.SessionButton ();
 
         var caps_lock_revealer = new Greeter.CapsLockRevealer ();
+        var num_lock_revealer = new Greeter.NumLockRevealer ();
```
Comment (torikulhabib) [inline src/Cards/UserCard.vala]: how about if we put grid in here.
i think like this.
        var notify_grid = new Gtk.Grid ();
        notify_grid.orientation = Gtk.Orientation.HORIZONTAL;
        notify_grid.halign = Gtk.Align.CENTER;
        notify_grid.margin_top = 4;
        notify_grid.column_spacing = 6;
        notify_grid.add (caps_lock_revealer);
        notify_grid.add (num_lock_revealer);

File: src/Cards/UserCard.vala [hunk 2]
```diff
@@ -49,13 +49,15 @@ public class Greeter.UserCard : Greeter.BaseCard {
...
 
         var password_grid = new Gtk.Grid ();
         password_grid.column_spacing = 6;
         password_grid.row_spacing = 6;
         password_grid.attach (password_entry, 0, 0);
         password_grid.attach (fingerprint_image, 1, 0);
         password_grid.attach (caps_lock_revealer, 0, 1, 2, 1);
+        password_grid.attach (num_lock_revealer, 1, 1, 2, 1);
```
Comment (torikulhabib) [inline src/Cards/UserCard.vala]: here change like this.
        password_grid.attach (notify_grid, 0, 1, 1, 1);

**Comment by torikulhabib** [comment]
screenshot from 2019-02-28 06-29-18

**Comment by torikulhabib** [comment]
screenshot from 2019-02-28 19-21-18

**Comment by cassidyjames** [comment]
@torikulhabib I thought you were suggesting large design changes like a fullscreen wallpaper, which would be far outside the scope of this issue. I think I see now that you're running greeter in a session. As @danrabbit mentioned, the horizontal layout of the warnings is probably good.

**Comment by cassidyjames** [comment]
Closing in favor of #265

**Pull request closed without merging**